test/fuzzing/request_proposal_fuzz.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ordering_service_fixture.hpp"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, std::size_t size) {
+  static fuzzing::OrderingServiceFixture fixture;
+
+  if (size < 4) {
+    return 0;
+  }
+
+  std::shared_ptr<OnDemandOrderingServiceImpl> ordering_service_;
+  std::shared_ptr<OnDemandOsServerGrpc> server_;
```

Comment:
Move both to fixture? (w/o init) Should improve performance a bit, since pointers allocated only once

Reply:
I don't think such a small performance improve worth it - a fixture user will have to think about additional initialisation.